astropy/utils/modal_utils.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = 'make_modal'
```

Comment:
Isn't `__all__` supposed to be a list or is this some tkinter magic?
```suggestion
__all__ = ['make_modal']
```

Reply:
oh good catch!